src/components/TOCSection.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import Card from "./reusable/Card";
+import CenteredCol from "./reusable/CenteredCol";
+import CustomSecondaryButton from "./reusable/CustomSecondaryButton";
+import styled from "styled-components";
+
+interface Props {
+  content: string[];
+}
+
+const TOCSection: React.FC<Props> = ({ content }): React.ReactElement => {
+  return (
+    <Card>
+      <div className="row">
+        <div className="col">
+          <h2>Table Of Content</h2>
+        </div>
+      </div>
+      <div className="col" style={{ margin: "-30px 0px 10px 0px" }}></div>
+      <div className="row">
+        <div className="col">
+          {content.map((line, i) =>
```

Comment:
Good code quality would be to abstract this function outside of the tree and name it getAlternateColorRow etc

Reply:
@ralph-dev Updated!

Reply:
@ralph-dev actually I made it a component rather than a function